packages/cells/src/cells/date-picker-cell.tsx

```diff
@@ -1,15 +1,89 @@
-import * as React from "react";
-import { CustomCell, CustomRenderer, drawTextCell, GridCellKind } from "@glideapps/glide-data-grid";
+import React from "react";
+import { assertNever } from "./common/support";
+import {
+    CustomCell,
+    CustomRenderer,
+    drawTextCell,
+    GridCellKind,
+    ProvideEditorCallback,
+} from "@glideapps/glide-data-grid";
 
 interface DatePickerCellProps {
```

Comment:
I think it would be awesome if we also support the `readonly` property for this cell. So, basically we add `readonly readonly?: boolean;` here and either 1) just set the input field to [readonly](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/readonly)  or 2) return a disabled [GrowingEntry](https://github.com/glideapps/glide-data-grid/blob/b470dee3845a817b32ad36d4d96c21f47547211e/packages/core/src/growing-entry/growing-entry.tsx#L16) (exposed as TextCellEntry) containing the `displayDate` (my preferred solution)

Reply:
I like option 2 better

Reply:
That's in the works! I like options 2 too! :)

Reply:
@jassmith , for the growingEntry, that's in the core package. Should I just copy over that package to the cells package?